meta-balena-common/recipes-bsp/grub/grub-conf.bb

```diff
@@ -0,0 +1,40 @@
+SUMMARY = "Grub configuration and other various files"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${RESIN_COREBASE}/COPYING.Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
+
+SRC_URI = " \
+    file://grub.cfg_external \
+    file://grub.cfg_internal_template \
+    file://grubenv \
+    "
+
+inherit deploy nopackages
+
+INHIBIT_DEFAULT_DEPS = "1"
+
+do_configure[noexec] = '1'
+do_compile() {
+    sed -e 's/@@TIMEOUT@@/3/' \
```

Comment:
If we are going to parametrize the timeout and not hardcode it in the grub config file it might be worth to have a Yocto variable with a default of 3s that could then be overriden by device integration layers if needed.